pkg/tracing/ebpf/caretta.bpf.c

```diff
@@ -170,7 +170,7 @@ int handle_tcp_data_queue(struct pt_regs *ctx) {
   return BPF_SUCCESS;
 };
 
-int handle_set_tcp_syn_sent(struct sock* sock) {
+static int handle_set_tcp_syn_sent(struct sock* sock) {
```

Comment:
Make it static inline :) I think that's what actually solves the issue (pretty sure clang optimizes it to be inline when declared static, but let's make it explicit)

Reply:
done